index.js

```diff
@@ -7,27 +7,10 @@ const Merge = require('broccoli-merge-trees');
 const fastbootTransform = require('fastboot-transform');
 const existsSync = require('exists-sync');
 
-let FONT_FILES = [
-  'Roboto-Thin.woff2',
-  'Roboto-Thin.woff',
-  'Roboto-Thin.ttf',
-  'Roboto-Light.woff2',
-  'Roboto-Light.woff',
-  'Roboto-Light.ttf',
-  'Roboto-Regular.woff2',
-  'Roboto-Regular.woff',
-  'Roboto-Regular.ttf',
-  'Roboto-Medium.woff2',
-  'Roboto-Medium.woff',
-  'Roboto-Medium.ttf',
-  'Roboto-Bold.woff2',
-  'Roboto-Bold.woff',
-  'Roboto-Bold.ttf'
-];
-
-function fontPath(app, name) {
-  return `${app.bowerDirectory  }/materialize/dist/fonts/roboto/${  name}`;
-}
+/**
+ * @type {string}
+ */
+const MATERIALIZE_JS_PATH = require.resolve('materialize-css');
```

Comment:
Need a "loud and clear" assertion if the path to `materialize-css` cannot be resolved for some reason.

Reply:
<summary>Here's a fancier version of the snippet I provided above
<details>

```javascript
'use strict';
const { dirname, resolve } = require('path');
const chalk = require('chalk');
try {
    const basePath = dirname(require.resolve('materialize-css/package.json'));
    const relativeSassFile = require('materialize-css/package.json').sassX;
    if (typeof relativeSassFile !== 'string') {
        throw new Error(`Found basePath (${basePath}) but package.json's 'sass' property was '${relativeSassFile}'.`);
    }
    const relativeSassPath = dirname(relativeSassFile);
    const fullSassPath = resolve(basePath, relativeSassPath);
    console.log(`basePath = ${basePath}\r\nrelativeSassPath = ${relativeSassPath}\r\nfullSassPath = ${fullSassPath}`);
} catch (e) {
    console.error(chalk.red('Failed to get SASS path from materialize-css.'),
                chalk.underline.bgBlue('(Is `materialize-css` installed?)\r\n'),
                chalk.yellow(e.stack)
    );
}
```

</details>
</summary>

Reply:
If `require.resolve` fails it will throw an error. Isn't that good enough?
```javascript
require.resolve('bogus')
```
throws
```
Error: Cannot find module 'bogus'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.resolve (internal/module.js:18:19)
```

Reply:
Also, note that `MATERIALIZE_JS_PATH` here will point to `.../materialize-css/dist/js/materialize.js`, which although technically is a *path*, many developers may think of a path as meaning a *directory* not a *file*. Above we used `MATERIALIZE_SASS_FOLDER` so I would recommend calling this one `MATERIALIZE_JS_FILE` for consistency.